interface/forms/misc_billing_options/new.php

```diff
@@ -42,97 +26,83 @@
+<span class=text><?php echo xlt('Box 10 D. EPSDT Referral Code');?> </span><input type=entry style="width: 25px;" size=2 name="medicaid_referral_code" value="<?php echo attr($obj{"medicaid_referral_code"});?>" >&nbsp;&nbsp;&nbsp;&nbsp;
 <label><span class=text><?php echo xlt('EPSDT'); ?> : </span><input type=checkbox name="epsdt_flag" value="1" <?php if ($obj['epsdt_flag'] == "1") echo "checked";?>></label><br><br>
-<span class="text" title="<?php echo xla("For HCFA 02/12 Onset date specified on the Encounter Form needs a qualifier");?>"></span>
-<span class=text title="<?php echo xla('For HCFA 02/12 Box 15 is Other Date with a qualifier to specify what the date indicates');?>"></span>
  <tr>
-  <td><span class=text><?php echo xlt('BOX 14. Is Populated from the Encounter Screen as the Onset Date');?>.</span></td>
+  <td><span class=text><?php echo xlt('Box 14.  Populated from the Encounter Screen as the Onset Date needs a qualifier.');?>
+    <?php generateDateQualifierSelect("box_14_date_qual",$box_14_qualifier_options,$obj); ?>
+  </span></td>
+ </tr><br><br>
+ <tr>
+   <td><span class=text><?php echo xlt('Box 15.  Other Date with a qualifier to specify what the date indicates.');?>
+     <?php $date_initial_treatment = $obj{"date_initial_treatment"}; ?>
+       <input type=text style="width: 70px;" size=10 class='datepicker' name='date_initial_treatment' id='date_initial_treatment'
+       value='<?php echo attr($date_initial_treatment); ?>'
+       title='<?php echo xla('yyyy-mm-dd'); ?>' />
+   </span></td>
+   <td>
+        <?php generateDateQualifierSelect("box_15_date_qual",$box_15_qualifier_options,$obj); ?>
+   </td>
+
  </tr><br><br>
  <tr>
-  <td><span class=text><?php echo xlt('BOX 16. Date unable to work from');?>:</span></td>
+  <td><span class=text><?php echo xlt('Box 16. Date unable to work from');?>:</span></td>
   <td><?php $off_work_from = $obj{"off_work_from"}; ?>
-    <input type=text style="width: 70px;" size=10 class='datepicker' name='off_work_from' id='off_work_from'
+    <input type=text style="width: 70px;" size=10 name='off_work_from' id='off_work_from'
     value='<?php echo attr($off_work_from); ?>'
-    title='<?php echo xla('yyyy-mm-dd'); ?>' />
+    title='<?php echo xla('yyyy-mm-dd'); ?>'
+    onkeyup='datekeyup(this,mypcc)' onblur='dateblur(this,mypcc)' />
+    <img src='../../pic/show_calendar.gif' align='absbottom' width='24' height='22'
+    id='img_off_work_from' border='0' alt='[?]' style='cursor:pointer'
+    title='<?php echo xla("Click here to choose a date"); ?>'></td>
   </td>
  </tr>
  &nbsp;&nbsp;
 <tr>
- <td><span class=text><?php echo xlt('BOX 16. Date unable to work to');?>:</span></td>
+ <td><span class=text><?php echo xlt('Box 16. Date unable to work to');?>:</span></td>
   <td><?php $off_work_to = $obj{"off_work_to"}; ?>
-    <input type=text style="width: 70px;" size=10 class='datepicker' name='off_work_to' id='off_work_to'
```

Comment:
(let me know if you want me to flag that file to add to the next 5.0.0 patch if it will make things easier)